pkg/render/node.go

```diff
@@ -320,6 +320,16 @@ func (c *nodeComponent) nodeCNIConfigMap() *v1.ConfigMap {
 		assign_ipv6 = "false"
 	}
 
+	var portmap string = ""
+	if c.cr.Spec.CalicoNetwork.HostPort != nil && *c.cr.Spec.CalicoNetwork.HostPort == operator.HostPortEnabled {
+		portmap = `,
+	{
```

Comment:
hm. the formatting here is pretty rough. Wonder if it would be nicer just to put this all on a single line, since it's obvious and harder to accidentally mess up with a spacing issue.

e.g., 

```
portmap = `,{"type": "portmap", "snat": true, "capabilities": {"portMappings": true}}`
```

WDYT?